packages/calcite-components/src/components/tile/tile.scss

```
@@ -68,6 +68,7 @@ calcite-link {
 
 .content-container {
   flex-direction: column;
+  word-break: break-word;
```

Can you use the word-break mixin here instead? For consistency.

```suggestion
@include word-break();
```